tiles-generation/docker/vector-tile/entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+#  Copyright OpenSearch Contributors
+#  SPDX-License-Identifier: Apache-2.0
+
+if [ "$1" == "generate" ]; then
```

Comment:
Shall we add help command to display supported options ?

Reply:
I plan to create a document file to explain usages

Reply:
Why not include this inside script only?

Reply:
Yes, will update